docs/projects/tcet-linux-website/frontend-components/verify-downloads.mdx

```diff
@@ -0,0 +1,222 @@
+
+2. `infoIconImg import:` This imports an ***image file*** `infoIcon.png` used for displaying an **information icon**.
+
+3. `CopyToClipboard import:` This imports **a component** named `CopyToClipboard` from a separate file. It is used to handle the ***copy-to-clipboard functionality***.
+
+     ```jsx title="VerifyDownloads.astro" {1-3} showLineNumbers
+        import clickToCopyImg from "../assets/clickToCopy.png";
+        import infoIconImg from "../assets/infoIcon.png";
+        import CopyToClipboard from "./CopyToClipboard";
+        // stepOne Component
+     ```
+
+### stepOne:
+
+`const stepOne` is an array. This array contains objects representing **different data** to be used after calling them. They contain the information for ***verifying downloaded*** files on **various operating systems**. Each object has properties such as **id**, **nameOfOS**, **description**, and **downloadableFileName** that hold specific information for each step.
+ 
+   ```jsx title="VerifyDownloads.astro" {2,4-6,8,10} showLineNumbers
+        // Import Component
+        const stepOne = [
+              {
+                id: 1,
+                nameOfOS: "Windows",
+                description:
+                  "Open windows terminal (powershell) in the Downloads folder or wherever the ISO is and type:",
+                downloadableFileName: "certutil -hashfile iso-name.iso sha256",
+              },
+              // Similarly more elements can be added
+            ];
+        // Content Section
+  ```
+
+## HTML Markup:
+
+The `HTML markup` represents the *visual structure* of the **verification steps and information**.
+It includes *headings*, *containers*, *buttons* and *elements* with specific classes for styling purposes.
+[***Content Section of Verify Section***](https://github.com/tcet-opensource/tcet-linux-website/blob/main/src/components/VerifyDownloads.astro#L31-#L116) can be *visited*.
+
+### Background & Title:
+
+1. `<div id="verify" class="bg-[#101828] p-4 px-10 lg:px-20">:` This `<div>` element has an `id attribute` set to **verify** and a class attribute with multiple classes. The classes define the *background color* and *padding for the section*.
+
+2. `<p class="font-title text-center text-[#FCFCFD] text-2xl md:text-4xl pt-2 pb-2">`  Verify Your Download`</p>:` This `<p>` element contains the heading ***"Verify Your Download."*** It has classes for font *styling*, *alignment*, and *text size*.
+
+  ```jsx title="VerifyDownloads.astro" {2,4,5,} showLineNumbers
+        // Import Component
+        <div id="verify" class="bg-[#101828] p-4 px-10 lg:px-20">
+                  <p
+                    class="font-title text-center text-[#FCFCFD] text-2xl md:text-4xl pt-2 pb-2" >
+                    Verify Your Download
+                  </p>
+            // Step 1 Section
+       </div>
+  ```
+
+### Step 1
```

Comment:
in step 1, step 2, verification explanation sections just explain what is important. most points discussed here are html and css which doesn't require too much detailing.